citation/caching.py

```diff
@@ -53,3 +58,35 @@ def _dictfetchall(cursor):
         dict(zip(columns, row))
         for row in cursor.fetchall()
     ]
+
+""" 
+    Method to cache the default distribution of publication across the year 
+    along with on which platform the code is made available information
+"""
+def initialize_publication_code_platform_cache():
+    data, platform = generate_publication_code_platform_data({}, RelationClassifier.GENERAL, "Publications")
+    cache.set("distribution_data", data, 86410)
```

Comment:
please set an appropriate namespace on the cache keys, e.g., citation:distribution_data or something along those lines - we may be sharing the cache with several other applications in the future